services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the `MovieSession.DoesNotExist` exception to prevent runtime errors if the `session_id` does not exist.